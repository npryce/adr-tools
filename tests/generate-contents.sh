adr new First Decision
adr new Second Decision
adr new Third Decision
adr generate toc
adr generate toc -r
