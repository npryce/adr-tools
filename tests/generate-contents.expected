adr new First Decision
doc/adr/0001-first-decision.md
adr new Second Decision
doc/adr/0002-second-decision.md
adr new Third Decision
doc/adr/0003-third-decision.md
adr generate toc
# Architecture Decision Records

* [1. First Decision](0001-first-decision.md)
* [2. Second Decision](0002-second-decision.md)
* [3. Third Decision](0003-third-decision.md)
adr generate toc -r
# Architecture Decision Records

* [3. Third Decision](0003-third-decision.md)
* [2. Second Decision](0002-second-decision.md)
* [1. First Decision](0001-first-decision.md)
